Snakefile

```diff
@@ -307,8 +307,17 @@ rule rnaspades:
     16
   shell:
     """
-    ##TODO = kmer shouldn't be fixed, & should be configurable from the beginning of the script (did run into some bugs with the auto parameter)
-    rnaspades.py --dataset {input.samples} -t {threads} -m {params.memory} -o rnaspades_out --only-assembler -k 47 &> {log}
+    myfiles=$(awk '{{print $3,$4}}' test_samples.txt | tr '\\n' ' ') 2>> {log}
```

Comment:
This code needs detailed comments to explain what is it actually supposed to do

Reply:
Done

Reply:
Not done! I mean detailed comments, explaining the purpose as well as the individual expressions.